subsys/net/lib/azure_fota/azure_fota.c

```diff
@@ -210,6 +210,9 @@ static bool parse_reported_status(const char *msg)
 	fw_obj = cJSON_GetObjectItemCaseSensitive(reported_obj, "firmware");
 	if (fw_obj == NULL) {
 		LOG_DBG("No 'firmware' object found");
+
+		report_needed = true;
```

Comment:
Unrelated change?

Reply:
It's intentional, in a separate commit: https://github.com/nrfconnect/sdk-nrf/pull/2962/commits/d831092bde9335e9798bb5a9d71b69cd4592a0f4
Kept it in the PR to not increase the CI load git such a small change.